repository/model/project_member.go

```diff
@@ -0,0 +1,29 @@
+package model
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type CreateProjectMemberInput struct {
```

Comment:
@bolshakoff In the end I think this naming feels appropriate for the repository layer as well. I'll update it in the other models.